packages/token-sdk/src/metadata/client/solanafm-client.ts

```diff
@@ -40,7 +40,10 @@ export class SolanaFmHttpClient implements SolanaFmClient {
     invariant(response.ok, `Unexpected status code fetching ${url}: ${response.status}`);
     const json = await response.json();
     invariant(isGetTokensResponse(json), "Unexpected SolanaFM getTokens response type");
-    invariant(json.status === "success", "Unexpected SolanaFM getTokens response status");
+    invariant(
+      json.status.localeCompare("success", undefined, { sensitivity: "base" }) === 0,
```

Comment:
should we only do `case` for sensistivity? I can't imagine the end point will return accented characters down the road

Reply:
So I dug into this, and here are the options for sensitivity.

`base`: Only strings that differ in base letters compare as different. Examples: "a" and "a" compare as same, "a" and "A" compare as same, "a" and "ä" compare as different.

`accent`: Only strings that differ in base letters or accents and other diacritic marks compare as different. Examples: "a" and "a" compare as same, "a" and "A" compare as same, "a" and "ä" compare as same.

`case`: Only strings that differ in base letters or case compare as different. Examples: "a" and "a" compare as same, "a" and "A" compare as different, "a" and "ä" compare as different.

`variant`: Strings that differ in base letters, accents and other diacritic marks, or case compare as different. Examples: "a" and "a" compare as same, "a" and "A" compare as different, "a" and "ä" compare as different.

SolanaFM is returning "Success" now instead of "success", so we just want a case insensitive comparison, and `case` actually evaluates "Success" !== "success"